src/service/config/configuration-service.ts

```diff
@@ -97,6 +99,26 @@ export class ConfigurationService {
     return this.configuration.parsedInputs.skipProjectExecution ? this.configuration.parsedInputs.skipProjectExecution.includes(node.project) : false;
   }
 
+  /**
+   * Checks whether given node needs to be checked out or not
+   * @param node
+   * @returns {Boolean} true if checking out needs to be skipped for the given node otherwise false
+   */
+  skipCheckout(node: Node): boolean {
+    if (this.configuration.parsedInputs.skipCheckout) {
+      return true;
+    }
+    return this.configuration.parsedInputs.skipProjectCheckout ? this.configuration.parsedInputs.skipProjectCheckout.includes(node.project) : false;
```

Comment:
This will not work. `skipCheckout` is boolean flag so its default value is always `false` and never `null` or `undefined`. So we will never be able to execute the right hand side of the `??` operator

Reply:
ahhh right!! you are right, sorry! I notice it in my first review but I forgot in the second one, sorry